script/mlperf-inference-v0.7.closed.image-classification/run.sh

```diff
@@ -1,23 +1,195 @@
 #!/bin/bash
 
-division="closed"
+# "closed" or "open" division
+division="open"
 task="image-classification"
 imagenet_size=50000
+# dry_run_mode="TRUE" for testing else "FALSE" for actual run.
+dry_run_mode="FALSE"
```

Comment:
Please do not hardcode, but allow reading from the environment variable like here:
https://github.com/dividiti/ck-zeromq/blob/master/script/explore-params/run.sh#L43

Reply:
Ah ookay